content/docs/reference-react-component.md

```diff
@@ -533,7 +533,7 @@ The second parameter to `setState()` is an optional callback function that will
 You may optionally pass an object as the first argument to `setState()` instead of a function:
 
 ```javascript
-setState(stateChange[, callback])
```

Comment:
this syntax indicates an optional argument, not an array

Reply:
@alexkrolick if that's the case in both cases then reject this PR.

Reply:
closing for now, but if you have a suggestion for an alternative way of showing the optional args, please submit another one!